virtual-organizations/LIGO.yaml

```
@@ -68,6 +68,9 @@ OASIS:
     - Name: Ian Harry
       DNs: /DC=org/DC=cilogon/C=US/O=LIGO/CN=Ian Harry ian.harry@ligo.org
       ID: bb0990ee72a93882c89747b1b6867758963a41a0
+    - Name: Gareth Davies
+      DNs: /DC=org/DC=cilogon/C=US/O=LIGO/CN=Gareth Davies gareth.davies@ligo.org
+      ID: c7d9a5cf10ecce28be62e62daf5a62d773f0185c:
```

```suggestion
      ID: c7d9a5cf10ecce28be62e62daf5a62d773f0185c
```

Oops, had a bad format in the previous suggestion. This should fix it